core/src/main/java/org/ehcache/core/InternalStatus.java

```diff
@@ -140,6 +140,13 @@ public InternalStatus to() {
     public boolean done() {
       return done != null;
     }
+
+    @Override
+    public String toString() {
+      return "to: " + to + "\n" +
```

Comment:
We shouldn't add `toString()` in such internal classes. Instead, in the CM, just output `status: getStatus()`